rnd/autogpt_server/autogpt_server/blocks/http.py

```diff
@@ -30,6 +30,8 @@ class Output(BlockSchema):
     def __init__(self):
         super().__init__(
             id="6595ae1f-b924-42cb-9a41-551a0611c4b4",
+            description="This block makes an HTTP request to the given URL.",
+            categories={BlockCategory.BASIC},
```

Comment:
Basic seems too generic to me - is there something more descriptive? Or we make the description of basic more clear?

Reply:
I really hope that the basic will only include these 2 blocks: HttpRequestBlock & ValueBlock. The others: PrintingBlock (this is purely testing block), ObjectLookup (should be removed once dynamic pins is implemented).